### ISSUE TITLE
Copy existing pytorch training code over from other project.

### ISSUE THREAD

**Issue by DallanQ** [rootsdev/nama#4]
Autoencoder + Triplet loss
